containers/dotnet/definition-manifest.json

```diff
@@ -7,6 +7,14 @@
 		"tags": [
 			"dotnet:${VERSION}-${VARIANT}"
 		],
+		"architectures": {
```

Comment:
Since you aren't excluding architectures for certain variants this can be simplified to:

`"architectures": ["linux/amd64", "linux/arm64"],`

Reply:
@bderusha , nice suggestion. Thanks.

Reply:
updated!